ext/tracer_tag_propagation/tracer_tag_propagation.c

```diff
@@ -44,9 +45,17 @@ void ddtrace_add_tracer_tags_from_header(zend_string *headerstr, zend_array *roo
             if (ZSTR_LEN(tag_name) >= sizeof("_dd.p.") &&
                 strncmp(ZSTR_VAL(tag_name), "_dd.p.", sizeof("_dd.p.") - 1) == 0) {
                 zval zv;
+                bool add = true;
                 ZVAL_STRINGL(&zv, valuestart, header - valuestart);
-                zend_hash_update(root_meta, tag_name, &zv);
-                zend_hash_add_empty_element(propagated_tags, tag_name);
+                if (strncmp(ZSTR_VAL(tag_name), DD_P_TS_KEY, sizeof(DD_P_TS_KEY) - 1) == 0) {
```

Comment:
Use `zend_string_equals_cstr()` instead of strcmp

Reply:
I changed for `zend_string_equals_literal`